config/environments/development.rb

```diff
@@ -58,5 +58,6 @@
   # Allow disabling the webpack dev server from the settings
   config.webpack.dev_server.enabled = SETTINGS.fetch(:webpack_dev_server, true)
   config.webpack.dev_server.https = SETTINGS.fetch(:webpack_dev_server_https, false)
-  config.hosts << SETTINGS[:fqdn]
+
+  config.hosts << SETTINGS[:hosts]
```

Comment:
This appends an array to an array. I think you wanted
```suggestion
  config.hosts += SETTINGS[:hosts]
```